gramex/__init__.py

```diff
@@ -270,12 +270,23 @@ def init(force_reload=False, **kwargs):
     # Services may return callbacks to be run at the end
     for key, val in final_config.items():
         if key not in conf or conf[key] != val or force_reload:
+
             if hasattr(services, key):
                 app_log.debug('Loading service: %s', key)
                 conf[key] = deepcopy(val)
+
+                # remove the comments in new (key, value) format
+                if isinstance(conf[key], dict):
+                    for comment_key in list(conf[key].keys()):
+                        if '_comment' in comment_key:
+                            app_log.debug('Removing comment (%s)', comment_key)
+                            conf[key].pop(comment_key)
+
```

Comment:
This is the perfect spot to remove the comment.

Should we rename `_comment` to `comment`? It's an exposed feature, not an internal one.

Less is better than more. Can't we just use: `if isinstance(conf[key], dict): conf[key].pop('comment', None)`?

Reply:
Doesn't work. Had started like this. Will discuss more when we connect tomorrow.